library/src/main/java/com/alamkanak/weekview/WeekViewEvent.java

```diff
@@ -193,11 +216,6 @@ public boolean equals(Object o) {
 
     }
 
-    @Override
```

Comment:
I think this method should remain. I guess you can return mId.hashCode() or something